aegis-web/yo/app/nav.html

```diff
@@ -103,7 +103,9 @@
         <!--ng-show="showDatasets"-->
         <form class="navbar-form">
           <div class="col-xs-12 col-sm-4 col-md-4 input-group">
-            <input type="text" class="form-control dark" style="height: 44px; background-color: #6baec6; border: none; color: #fff"
+
+            <input type="text" class="form-control dark" style="height: 44px; background-color: #ffffff; border: none; border-top-left-radius: 10px; border-bottom-left-radius: 10px;"
```

Comment:
Is this the desired result? 
![image](https://user-images.githubusercontent.com/1640100/52804123-b6f6fe00-3083-11e9-9050-659b83ff71b9.png)

Reply:
Yes, but if you prefer another style tell me